circle.yml

```diff
@@ -3,17 +3,30 @@ general:
     ignore:
       - gh-pages # list of branches to ignore
 machine:
+  environment:
+    YARN_VERSION: 0.20.3
+    PATH: "${PATH}:${HOME}/.yarn/bin:${HOME}/${CIRCLE_PROJECT_REPONAME}/node_modules/.bin"
   node:
-    version: 6.2.0
+    version: 4
```

Comment:
Can you test that the website builds using Node 4? Last I checked, we do require 6 to generate the site.